build.gradle

```diff
@@ -37,6 +151,11 @@ configure(opentelemetryProjects) {
     sourceCompatibility = 1.7
     targetCompatibility = 1.7
 
+    java {
+        withJavadocJar()
```

Comment:
Gradle has finally supported these natively for a few versions. It's nice since maven publishing will pick them up automatically too without the `artifact` stuff